### ISSUE TITLE
Add back button

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-desktop#208]
In a browser, I find hitting alt-left (the keyboard shortcut for the back button) super handy and do it all the time.

We should have a back button (and maybe a forward button? but not sure where to put it), and also the hotkeys alt-left/right to go back and forward. Or Cmd-left/right, or whatever the appropriate thing is per platform.

**Comment by timabbott** [comment]
Probably worth separating this issues.  `alt+left` and `alt+right` we should be able to dispatch quickly without making any decisions, but some design work needs to be done to figure out where to put the back/forward buttons.

**Comment by akashnimare** [comment]
> In a browser, I find hitting alt-left (the keyboard shortcut for the back button) super handy and do it all the time.

@gnprice added `Alt+Left` and `Alt+Right`. 

macOS -

![image](https://user-images.githubusercontent.com/2263909/28189163-77c2d354-6842-11e7-940a-e72c51edecd3.png)

Ubuntu - 

![image](https://user-images.githubusercontent.com/2263909/28189454-9aae9000-6843-11e7-80c8-7bb479feb485.png)

**Comment by timabbott** [comment]
Nice!  We may want to close this and open a new, somewhat lower priority issue for where to put a more visible back button (or are we still waiting on Windows?).

**Comment by gnprice** [comment]
@akashnimare Sweet! Thanks for the fast action :)

On macOS, will Cmd-left and Cmd-right work, as well as Cmd-[ and Cmd-]? I don't currently have a Mac to try this on, but I used them for years, and I could not have told you that Cmd-[ / Cmd-] were browser shortcuts for back/forward; the shortcuts I used were I'm pretty sure Cmd-left and Cmd-right.

**Comment by akashnimare** [comment]
Yeah, normally browsers have both the shortcuts available `Cmd-[/Cmd-]` and `Cmd-Left/Cmd-right`. I have changed it to `Cmd-Left/Cmd-Right` since it's the expected shortcuts. @timabbott on windows it works fine.

![shortcut](https://user-images.githubusercontent.com/2263909/28241148-78dfb926-69ac-11e7-96ee-c3555958c1ac.PNG)

**Comment by timabbott** [comment]
Great.  If we can make both versions on Mac work to match browser behavior, that'd be nice, but probably not super important.

**Comment by akashnimare** [comment]
@timabbott done.

**Comment by abhigyank** [comment]
I think the back button can go on the sidebar, above the icons.
![screenshot from 2018-01-21 12-56-10](https://user-images.githubusercontent.com/20434085/35191824-ac217886-feaa-11e7-96b6-0594ae5c3ba9.png)
In the above picture, I would say above the Zulip icon. What do you guys think?

**Comment by abhigyank** [comment]
A mockup for the back button position. Looks good? @akashnimare  @timabbott  @gnprice @cPhost 
![screenshot from 2018-02-12 22-11-47](https://user-images.githubusercontent.com/20434085/36108076-e68729c8-1041-11e8-8a3d-e0b4f73c1950.png)

**Comment by akashnimare** [comment]
Personally, I don't think that the back button is necessary. I have not seen a back button in any other chat/electron app. @timabbott what do you think?

**Comment by timabbott** [comment]
I personally find that I use the arrow key version of the "back" button pretty often for navigation.  I suspect that suggests that folks who use the mouse would find the "back" button useful as well.  I think we should post on #design asking whether folks use the back button in the Zulip webapp (which is generally very well integrated).  If we find they do, we might want to support it in the desktop app as well.  The space is certainly cheap.

**Comment by abhigyank** [comment]
That is a good thought @timabbott . 
I will make a PR which we can merge/close as needed after discussion on #design.
